tests/pass/tree_borrows/unique.rs

```diff
@@ -1,7 +1,6 @@
-//@revisions: default uniq
+//@revisions: default
```

Comment:
This doesn't need to use revisions any more.

Reply:
Presumably this test should also be removed? There is no special handling for `Unique` any more so nothing needs to be tested.

Reply:
Yeah makes sense.